rel/files/mongooseimctl

```diff
@@ -219,8 +239,8 @@ done
 
 case $1 in
     'start') start;;
-    'add_to_cluster') add_to_cluster;;
-    'remove_from_cluster') remove_from_cluster;;
+    'join_cluster') join_cluster;;
```

Comment:
Why this name change? I understand its better but I see no updated in the doc about this change.